src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -11058,15 +11058,17 @@ public sealed partial class AmbiguousMatchException : System.SystemException
     }
     public abstract partial class Assembly : System.Reflection.ICustomAttributeProvider, System.Runtime.Serialization.ISerializable
     {
+        internal const string ThrowingMessageInRAF = "This member throws an exception for assemblies embedded in a single-file app";
```

Comment:
This won't be preserved next time somebody tries to auto-generate System.Runtime https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md . 

These ref files should use inline strings since it is what the auto-generator produces.